hpx/parallel/executors/auto_chunk_size.hpp

```diff
@@ -55,6 +55,12 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
           : min_time_(rel_time.value().count())
         {}
 
+        template <typename Executor>
+        static bool variable_chunk_size(Executor&)
+        {
+            return false;
+        }
```

Comment:
Yes, but I can't use traits directly, I think we have discussed already on Thursday afternoon? I merged variable_chunk_size and get_chunk_size into one executor parameter type called chunk_size, hence this function has to be defined for unwrapper. Otherwise I'd need again another tag for variable_chunk_size.